### PR TITLE
perf(WebGL): rework WebGL volumeMapper interactive rendering

### PR DESCRIPTION
**Pull request by martinken**
Update the renderwindowinteractor to set the recentFrameTime
from the new approach which uses at least a one second
timeframe to compute rendering rates.

Update the WebGL VolumeMapper to use a callback for when the
animation rate changes on the renderwindowinteractor
and simplify a bit how the use of the smaller viewport
is handled.

Adjust the volumeviewer example to be 30 fps because 15fps
feels slow.

### CODE REVIEW

File: Sources/Rendering/OpenGL/VolumeMapper/index.js [hunk 1]
```diff
@@ -947,107 +947,84 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
...
-    // if (2.0 * diag / (newSize[0] + newSize[1]) < 0.25 * model.renderable.getSampleDistance()) {
-    //   model.renderable.setSampleDistance(0.25 * model.renderable.getSampleDistance());
-    // }
+    // use/create/resize framebuffer if needed
+    if (model._useSmallViewport) {
+      const size = model.openGLRenderWindow.getFramebufferSize();
 
-    // create/resize framebuffer if needed
-    if (xyf > 1.43) {
+      // adjust viewportSize to always be at most the dest fo size
```
Comment (floryst) [inline Sources/Rendering/OpenGL/VolumeMapper/index.js:1012]: I assume this is meant to be "dest fb size"?

**Review by floryst — APPROVED** [review]
LGTM

File: Sources/Rendering/OpenGL/VolumeMapper/index.js (hunk 1 above)
Comment (martinken) [reply Sources/Rendering/OpenGL/VolumeMapper/index.js:1012]: fo = framebuffer object

Comment (martinken) [reply Sources/Rendering/OpenGL/VolumeMapper/index.js:1012]: But yes, I use framebuffer above when it should technically be framebuffer texture

**Pull request merged**